postgresql/mingw-w64/PKGBUILD

```diff
@@ -43,7 +46,7 @@ build() {
     ln -s "${srcdir}/postgresql-$pkgver/src/interfaces/ecpg/pgtypeslib/libpgtypesdll.def" src/interfaces/ecpg/pgtypeslib/
     ln -s "${srcdir}/postgresql-$pkgver/src/interfaces/ecpg/compatlib/libecpg_compatdll.def" src/interfaces/ecpg/compatlib/
 
-    make
+    make -j4
```

Comment:
Please don't. Modify your `makepkg.conf` to customize those options.

Reply:
OK